openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -0,0 +1,483 @@
+              description: Price description of the invoice item.
+              example: 50% of the markup for approved transactions
+              maxLength: 255
+            unitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            quantity:
+              description: Quantity of the invoice item.
+              type: integer
+            period:
+              description: Date interval of the invoice item.
+              type:
+                - 'string'
+                - 'null'
+            setupUnitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialUnitPrice:
+              description: Unit price of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialPeriod:
+              description: Date interval of the invoice item trial.
+              type:
+                - 'string'
+                - 'null'
+            taxAmount:
+              description: Tax amount of the invoice item.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            setupTaxAmount:
+              description: Tax amount of the setup that is applied to the invoice.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+            trialTaxAmount:
+              description: Tax amount of the trial that is applied to the invoice.
+              type:
+                - 'number'
+                - 'null'
+              format: double
+  paymentTerms:
+    description: Payment terms for the customer which are displayed on the quote.
+    type: string
+  expirationTime:
+    description: |-
+      Date and time when the quote expires. The default expiration time is one month from the time the quote is issued.
+      In a `draft` state, this field may be `null`.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+  issuedTime:
+    description: Date and time when the quote is issued.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  acceptedTime:
+    description: Date and time when the quote is accepted.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  rejectedTime:
+    description: Date and time when the quote is rejected.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  canceledTime:
+    description: Date and time when the quote is canceled.
+    type:
+      - 'string'
+      - 'null'
+    readOnly: true
+    format: date-time
+  redirectUrl:
+    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
```

nitpick: [line breaks](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#line-breaks)
```suggestion
    description: |-
    URL to redirect the customer to when a quote is rejected. 
    The default value is the website URL.
```